_posts/2022-08-05-crossplane-demo.md

```diff
@@ -0,0 +1,231 @@
+---
+layout:      posts
+background:  shortBackground
+title:       "Crossplane, a Kubernetes-powered IaC Tool"
+subtitle:    ""
+date:        2022-08-05 09:00:00
+author:      "Marty Henderson"
+description: ""
+headerImg:  "/assets/images/posts/2022-08-05-crossplane/popsicle-header.jpg"
+cross-post-link: "https://nalth.is/crossplane"
+cross-post-text: "Nalthis"
+---
+
+
+Header Image - Photo by <a href="https://unsplash.com/es/@taylorheeryphoto?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Taylor Heery</a> on <a href="https://unsplash.com/s/photos/popsicle?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
```

Comment:
Can you remove the utm parameters? We don't need to pass through tracking data.

This isn't the header image despite the var name, it is actually the thumbnail on the blog post list. Not sure if you want to rephrase that.

Reply:
True, in my blog post it was the header, I'll just say "image" and drop the tracking.